client/offchain/src/api.rs

```diff
@@ -119,19 +108,73 @@ impl<Storage: OffchainStorage> OffchainExt for Api<Storage> {
 		old_value: Option<&[u8]>,
 		new_value: &[u8],
 	) -> bool {
+		log::debug!(
```

Comment:
For these kind of logging, I can really recommend `tracing::debug!/*` in the future. They provide some nice "structured logging"

General comment, a target would be nice.